vuepress/docs/next/tutorials/curate/bundle-private-images.md

```diff
@@ -0,0 +1,85 @@
+---
+sidebarDepth: 2
+---
+
+# Install Bundle from a Private Image Registry
+
+This tutorial provides to way to utilize bundles from a private image repository in your Entando projects. The steps below use environment variables to pass the Secret for authentication required by private repositories.
+
+**New or Existing Bundle**
+* If you have an existing bundle in a private image repository, start at [Step 1: Create the Registry Credentials](#step-1-create-the-registry-credentials). 
+* For a new project, start at [Step 2: Create and Apply the Secret](#step-2-create-and-apply-the-secret).
+
+## Prerequisites
+
+* [A working instance of Entando](../../../docs/getting-started/)
+* Verify dependencies with the [Entando CLI](../../docs/getting-started/entando-cli.md#check-the-environment): `ent check-env develop`
+
+## Tutorial
+### Step 1: Create the Registry Credentials
+1. Create the registry JSON configuration using your registry and credentials on the port of your choice: 
+``` json
+{
+  "auths": {
+    "YOUR-REGISTRY.com": {
+        "username": "YOUR-USERNAME",
+        "password": "YOUR-PASSWORD"
+    }
+  }
+}
+```
+
+2. [Convert the JSON configuration into a base64 string](https://www.base64encode.org/)  
+
+### Step 2: Create and Apply the Secret
+1. Create `container-registry-secret.yaml` in your namespace with the following snippet. Replace the registryCredentials value with your own. 
+``` yaml
+kind: Secret
+apiVersion: v1
+metadata:
+   name: container-registry-secret
+type: Opaque
+data:
+   registryCredentials: "ewrCoMKgImF1dGhzIjogewrCoMKgwqDCoCJyZWdpc3RyeS5odWIuZG9ja2VyLmNvbS9qeXVubWl0Y2hlbGwvOjgwODUiOiB7CsKgwqDCoMKgwqDCoMKgwqAidXNlcm5hbWUiOiAianl1bm1pdGNoZWxsIiwKwqDCoMKgwqDCoMKgwqDCoCJwYXNzd29yZCI6ICJKeW0xMTIyMzM9IgrCoMKgwqDCoH0KwqDCoH0KfQ=="
```

Comment:
@jyunmitch please replace with a bogus string. This may have real credentials in it.